### ISSUE TITLE
Add option for numeric/alphanumeric code generation

### ISSUE THREAD

**Issue by codingquark** [Oviyum/laravel-otp#3]
Currently, we generate only numeric codes. Add option to switch between numeric and alpha-numeric codes.

**Comment by donmbelembe** [comment]
Your migration table says **token is an integer**
`$table->integer('token')->unsigned();`
So, we must update it to string in other to accept numeric/alphanumeric

**Comment by codingquark** [comment]
I agree. I was also thinking about the API to be exposed for the application to decide which kind of token it wants.

A simple config file setting should work, but open to better ideas.

**Comment by donmbelembe** [comment]
Of course but either numeric or aphanumeric tokens will use the same column on the database

**Comment by codingquark** [comment]
So, what point are you trying to make?

**Comment by donmbelembe** [comment]
i'm working on this enhancement and i've changed the token column datatype to string. i would like to know if that's the right way
